src/dir.cr

```diff
@@ -93,6 +96,24 @@ class Dir
     String.new(dir).tap { C.free(dir as Void*) }
   end
 
+  def self.chdir path
+    if C.chdir(path) != 0
+      raise Errno.new("Error while changing directory")
+    end
+  end
+
+  def self.chdir(path)
+    old = working_directory
+    chdir(path)
+    yield
```

Comment:
Thanks! I thought maybe that's the case after I wrote the comment. 
A part of me hoped it wouldn't be because it's so easy to miss without looking at the implementation of the method.
Not that great for documentation. Thank you =)